### ISSUE TITLE
Error Installing with Yarn

### ISSUE THREAD

**Issue by fresh5447** [bigcommerce/checkout-sdk-js#377]
I am using node version `8.11.1`

When I attempt to install the package: `yarn add @bigcommerce/checkout-sdk`'

```
$ yarn add @bigcommerce/checkout-sdk
yarn add v1.6.0
warning ../../../package.json: No license field
[1/4] 🔍  Resolving packages...
warning @bigcommerce/checkout-sdk > messageformat > glob > minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
[2/4] 🚚  Fetching packages...
error @bigcommerce/script-loader@0.1.5: The engine "node" is incompatible with this module. Expected version "^6.10.0".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Comment by davidchin** [comment]
@fresh5447 thanks for raising this. It seems we an `engine` field in our dependencies that shouldn't be there. Yarn seems to have a strict engine check that's why you can't install the package. I think for now, as a work around, you might want to try doing `yarn add @bigcommerce/checkout-sdk --ignore-engines`

**Comment by davidchin** [comment]
This should be fixed in [v1.1.1](https://github.com/bigcommerce/checkout-sdk-js/releases/tag/v1.1.1).
